web-ng/src/app/components/user-profile-popup/user-profile-popup.component.scss

```diff
@@ -42,10 +45,24 @@
 
 .user-description {
   margin: 0px auto;
-  margin-top: 10px ;
+  margin-top: 10px;
   text-align: center;
 }
 
 .btn-signout {
   margin-top: 20px;
 }
+
+[data-letters]:before {
```

Comment:
Duplicated UI widgets should implemented as a separate component (e.g., user-avatar-component).

Reply:
Done..